os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/da_ble_peripheral/ble_tizenrt_da_server.c

```diff
@@ -579,10 +761,10 @@ da_ble_result_type rtw_ble_server_delete_bonded_device(uint8_t bond_addr[DA_BLE_
     }
 
     da_ble_result_type ret = DA_BLE_RESULT_TYPE_FAILURE;
-    T_TIZENRT_DELETE_BOND_PARAM *param = os_mem_alloc(0, sizeof(T_TIZENRT_DELETE_BOND_PARAM));
+    T_TIZENRT_SERVER_DELETE_BOND_PARAM *param = os_mem_alloc(0, sizeof(T_TIZENRT_SERVER_DELETE_BOND_PARAM));
```

Comment:
Could you check that `os_mem_alloc` tries to allocate the memory into kernel heap?

Reply:
@JeonginKim 
I think 'os_mem_alloc' is using kernel mem alloc because this implement is in kernel side.
Does it make any issues?

Reply:
If it allocates the memory into kernel heap, then ok. No issue.